pkg/controller/clusters/clusterapi.go

```diff
@@ -37,3 +38,11 @@ func CheckControlPlaneReady(ctx context.Context, client client.Client, log logr.
 	log.Info("CAPI control plane is ready")
 	return controller.Result{}, nil
 }
+
+func CapiClusterObjectKey(cluster *anywherev1.Cluster) client.ObjectKey {
```

Comment:
nit: add doc comment?
```suggestion
// CapiClusterObjectKey generates and ObjectKey for the CAPI cluster owned by
// the provided eks-a cluster
func CapiClusterObjectKey(cluster *anywherev1.Cluster) client.ObjectKey {
```